test/integration/features/story_registry.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Local image to image-registry to deployment
+
+    The user creates a local container image with an app. They then
+    push it to the openshift image-registry in their
+    project/namespace. They deploy and expose the app and check its
+    accessibility.
+
+    Scenario Outline: Start CRC
+        Given executing "crc setup" succeeds
+        When starting CRC with default bundle and hypervisor "<vm-driver>" succeeds
+        Then stdout should contain "CodeReady Containers instance is running"
+        And executing "eval $(crc oc-env)" succeeds
+        And executing "oc login --insecure-skip-tls-verify -u kubeadmin -p 9gZtH-EWZLT-mZNFP-bkKwm https://api.crc.testing:6443" succeeds
+        
+    @darwin
+        Examples:
+            | vm-driver  |
+            | hyperkit   |
+
+    @linux
+        Examples:
+            | vm-driver |
+            | libvirt   |
+
+    @darwin @linux @windows
+    Scenario: Create local image
+        Given executing "cd ../../../testdata" succeeds
+        When executing "sudo podman build -t hello:test ." succeeds
+        And executing "cd ../integration" succeeds
+        Then executing "sudo podman images" succeeds
```

Comment:
On host like `darwin` and `windows` there is no podman available so this test can be only run on linux